spec/factories/tools.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :tool do
+    title { generate(:tool_title) }
+    description
+    total { [2, 4, 7].sample }
```

Comment:
Do you want the test fixture to change randomly? If so, why not just use rand(1..1000) or similar.